spec/models/child_benefit_tax_calculator_spec.rb

```diff
@@ -556,7 +556,7 @@
         children_count: 1,
         starting_children: {
           "0" => {
-            start: { day: "24", month: "06", year: "2013" },
+            start: { day: "01", month: "07", year: "2013" },
```

Comment:
Not sure why you have changed the day and the month for this test. 
Could you please explain it in the commit notes?